services/src/api/handlers/datasets.rs

```diff
@@ -1473,24 +1289,111 @@ pub async fn list_volume_file_layers_handler<C: ApplicationContext>(
     Ok(web::Json(VolumeFileLayersResponse { layers }))
 }
 
+/// Creates a new dataset using files available on the volumes.
+/// The created dataset will be visible to all users.
+/// Requires an admin session.
+#[utoipa::path(
+    tag = "Datasets",
+    post,
+    path = "/dataset/public", 
```

Comment:
what about the regular dataset handler with post, path = "/dataset"? You deleted it but this one does not replace it.

Reply:
Ich verstehe das mit `/dataset/public` nicht, das gab es irgendwie nie.

Reply:
yes, the web resource path and the utoipa path annotation for create_dataset are conflicting. I guess you can just change the utoipa annotation to fix the generated client?

Reply:
[805e75f](https://github.com/geo-engine/geoengine/pull/1006/commits/805e75f8a058e57c27547e66765fb2eee6984c5a)